assets/female_avatar.svg

```diff
@@ -0,0 +1 @@
9-12.21s3.99,0,13.14,6.66a130.9934,130.9934,0,0,1,14.3,12.34C828.69,697.93,828.94,698.17,829.19,698.42Z" transform="translate(-262 -112)" fill="#d0cde1"/><path d="M828.49,697.74a11.39223,11.39223,0,0,1-.56,1.84,337.81887,337.81887,0,0,1-27.65,22.7,117.95015,117.95015,0,0,1-25.79,6.13s.75,3.76,1.43,10.26q-3.93,2.4-7.93,4.69-5.22,3-10.57,5.81c-1.44.77-2.88995,1.51-4.34,2.25-2.12,1.09-4.25,2.14-6.4,3.17-.37.18-.74.36-1.11.53-1.4.68-2.82,1.34-4.24,2a334.89638,334.89638,0,0,1-57.46,20.39c-.11.03-.21.05-.32.08-1.29.33-2.59.65-3.89.96-.97.24-1.95.47-2.93.69a2.48782,2.48782,0,0,1-.37.09c-2.01.47-4.02.92-6.05,1.33-1.14.24-2.28.48-3.42.71-.69.15-1.39.28-2.08.41-.87.17-1.73.33-2.6.5-.48.1-.96.18-1.45.26-1.1.2-2.2.4-3.29.59-1.86.32-3.74.63-5.61.91-1.36.21-2.71.41-4.07.6-1.6.23-3.2.44-4.81.64-.44.07-.88.12-1.32.17q-.85491.10492-1.71.21c-1.15.14-2.3.27-3.46.39-.6.07-1.19.13-1.79.19q-1.71.18-3.42.33c-1.47.13995-2.94.26-4.42.38-1.8.14-3.59.27-5.4.38-1.16.07-2.32.14-3.49.2-2.98.16-5.96.27-8.96.35-1.49.04-2.99.07-4.49.09h-.03c-1.49.02-2.99.03-4.49.03a339.151,339.151,0,0,1-45.45-3.04q-16.575-2.22-32.62-6.02c-.43-.1-.86-.2-1.28-.31-1.08-.25-2.15-.52-3.22-.79-.95-.24-1.89-.49-2.84-.74a335.35735,335.35735,0,0,1-70.6-27.19c-.11-.06-.23-.12-.34-.18-.65-.34-1.29-.67-1.93-1.02,54.89-12.79,141.63-32.96,216.9-50.24q3.165-.72,6.29-1.44c27.8-6.38,53.84-12.32,75.87006-17.29q3.42-.765,6.69995-1.5c23.67-5.32,42.05005-9.36,52.02-11.38a81.48836,81.48836,0,0,1,8.14-1.43l20.81,32.25Z" transform="translate(-262 -112)" fill="#2f2e41"/><circle cx="161.4491" cy="205.85142" r="48.91333" fill="#2f2e41"/><circle cx="461.91668" cy="191.87618" r="48.91333" fill="#2f2e41"/><path d="M671.50954,282.91333a48.91227,48.91227,0,0,1,43.67262-48.62417,48.91338,48.91338,0,1,0,0,97.24833A48.91227,48.91227,0,0,1,671.50954,282.91333Z" transform="translate(-262 -112)" fill="#2f2e41"/><path d="M368.796,357.41647a48.91227,48.91227,0,0,1-2.83554-65.296,48.91337,48.91337,0,1,0,68.06011,69.46264A48.91226,48.91226,0,0,1,368.796,357.41647Z" transform="translate(-262 -112)" fill="#2f2e41"/><path d="M573.68289,359.77712c-57.88732,0-104.81427,27.37407-104.81427,61.14166,0,15.10906,9.43269,28.91775,25.003,39.58556l20.1779-14.62031-5.48723,22.90446c2.27682,1.05513,4.60162,2.07636,7.02643,3.02019l30.1437-35.16776v43.20369a175.95757,175.95757,0,0,0,27.95047,2.21583c57.88732,0,104.81427-27.37407,104.81427-61.14166S631.57021,359.77712,573.68289,359.77712Z" transform="translate(-262 -112)" fill="#2f2e41"/><path d="M675.00335,303.87618s-7.59666,34.93809,39.87428,48.91333Z" transform="translate(-262 -112)" fill="#2f2e41"/><path d="M468.86862,310.23519s17.469,39.06051-34.93809,56.52955Z" transform="translate(-262 -112)" fill="#2f2e41"/><ellipse cx="489.41344" cy="514.60154" rx="8.28703" ry="19.88888" transform="translate(-382.60837 38.98959) rotate(-15.63946)" fill="#d0cde1"/><ellipse cx="688.30226" cy="498.02747" rx="19.88888" ry="8.28703" transform="translate(-113.09381 1042.41397) rotate(-86.47577)" fill="#d0cde1"/></svg>
```

Comment:
I would recommend to use image link instead of SVG for now.
placeimg is very helpfull to get images with any desired diensions.
Example: https://placeimg.com/80/80/animals

Reply:
Thanks Prajwal, will look into it